examples/basic-export/package.json

```diff
@@ -2,8 +2,8 @@
   "private": true,
   "scripts": {
     "dev": "next",
-    "build": "next build && next export",
-    "start": "next start"
+    "build": "next build",
+    "start": "serve out"
```

Comment:
Should `next start` just serve the `out` folder when `output: 'export'` is configured 🤔

Reply:
Maybe but then there would be no reason to export. I think the purpose of using another pkg here is to demonstrate it works without next